pybossa/view/bloomberg.py

```diff
@@ -83,11 +83,13 @@ def handle_bloomberg_response():
             # User is authenticated on BSSO, but does not yet have a GIGwork account, auto create one.
             user_data = {}
             try:
-                user_data['fullname']   = attributes['firstName'][0] + " " + attributes['lastName'][0]
-                user_data['email_addr'] = attributes['emailAddress'][0]
-                user_data['name']       = attributes['username'][0]
-                user_data['data_access']= ["L4"]
-                user_data['password']   = generate_password()
+                user_data['fullname']    = attributes['firstName'][0] + " " + attributes['lastName'][0]
+                user_data['email_addr']  = attributes['emailAddress'][0]
+                user_data['name']        = attributes['username'][0]
+                user_data['password']    = generate_password()
+                user_data['metadata']    = {"admin" : "BSSO"}
```

Comment:
if user_data['metadata'] contains other fields, this would overwrite, can rephrase to:
`user_data['metadata'] ["admin"] ="BSSO"`

Reply:
Hmm realized this is for creating new user, so ` user_data['metadata']    = {"admin" : "BSSO"}` is correct.

Reply:
Oh good! It was going to get a little more complicated otherwise.